lib/resolvers.js

```diff
@@ -64,16 +64,18 @@ const wrapResolvers = function (bind, resolvers = {}) {
         wrapped[name] = {};
       }
 
-      for (const [resolverName, func] of Object.entries(value)) {
+      for (const [resolverName, funcOrPrimitive] of Object.entries(value)) {
         if (wrapped[name][resolverName]) {
           continue;
         }
         if (['Query', 'Mutation', 'Subscription'].indexOf(name) > -1) {
           debug(`memoized ${name}.${resolverName}`);
-          wrapped[name][resolverName] = memoize(name, resolverName, func.bind(bind));
+          wrapped[name][resolverName] = memoize(name, resolverName, funcOrPrimitive.bind(bind));
           continue;
         }
-        wrapped[name][resolverName] = func.bind(bind);
+        // bind if the value mapped to a resolverName is a function
+        // otherwise dont to support internal enum value remapping
+        wrapped[name][resolverName] = funcOrPrimitive instanceof Function ? funcOrPrimitive.bind(bind) : funcOrPrimitive;
```

Comment:
Use `typeof func === 'function'` instead

Reply:
Specifically because you are checking for a function primitive not a function object. Use `typeof` to check primitives.

Reply:
Makes sense. Wrapping my head around when to use what (`typeof` vs `instanceof`) seems like it's suggested that you use `typeof` for built in types and `instanceof` for custom types? I'll make the change